app/assets/javascripts/components/common/wiki_overview_stats.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import WikiOverviewTabs from './OverviewStats/wiki_overview_tabs';
+import NamespaceOverviewStats from './OverviewStats/namespace_overview_stats';
+import WikidataOverviewStats from './wikidata_overview_stats';
+
+import ArticleUtils from '../../utils/article_utils';
+
+
+const WikiOverviewStats = ({ wikidataStats, wikiNamespaceStats }) => {
```

Comment:
I think this component's name could be improved, to more clearly indicate what it's for (displaying and interacting with a set of tabs that show narrow wiki- or namespace-specific stats).

Reply:
`OverviewStatsTabs`?

"overview" identifies it a component of the Overview tab, and "stats tabs" identifies it as the tabbed stats that show up separately from the static overall stats at the top.

Reply:
Alright.